src/gemini_computer_use/main.py

```diff
@@ -65,19 +72,36 @@ def main():
     prompt += f"\nGoal: {goal}"
 
     messages = [prompt]
-    for i in range(5):
+    def is_goal_completed():
+        messages = [f"Goal: {goal}. Send <finish> if the goal is completed. Else explain why it is not completed."]
+        messages.append(computer.take_screenshot())
         response = client.models.generate_content(
             model='gemini-2.0-flash-exp', contents=messages
         )
-
-        if "<screenshot>" in response.text:
-            print(response.text)
+        response_text = response.text
+        if "finish" in response_text:
+            return True, "Goal is completed."
+        return False, response_text
```

Comment:
Did you see improvements with this double check for goal completion vs just asking for a <finish> tag?

Reply:
![image](https://github.com/user-attachments/assets/3c6f88c8-7405-46fd-83b9-1e259e5c9452)

Since the terminal is active, alt f is not worked. Even the goal completion also gave wrong results even with `gemini-exp-1206`.